wagtail/admin/action_menu.py

```diff
@@ -147,27 +162,43 @@ def __init__(self, request, **kwargs):
         self.request = request
         self.context = kwargs
         self.context['user_page_permissions'] = UserPagePermissionsProxy(self.request.user)
+        self.default_item = SaveDraftMenuItem(_isDefault=True)
 
         self.menu_items = [
             menu_item
             for menu_item in _get_base_page_action_menu_items()
             if menu_item.is_shown(self.request, self.context)
         ]
 
-        self.menu_items.sort(key=lambda item: item.order)
+
+        self.menu_items.sort(key=lambda item: (item._isDefault, item.order))
 
         for hook in hooks.get_hooks('construct_page_action_menu'):
             hook(self.menu_items, self.request, self.context)
 
+        # Chcek improperly configured default menu_itmes
+        default_count = 0
+        for menu_item in self.menu_items:
+            if menu_item._isDefault:
+                self.default_item = menu_item
+                default_count += 1
+
+            if default_count > 1:
+                raise ImproperlyConfigured("Only one menu item can be default")
+
     def render_html(self):
         return render_to_string(self.template, {
             'show_menu': bool(self.menu_items),
             'rendered_menu_items': [
                 menu_item.render_html(self.request, self.context)
                 for menu_item in self.menu_items
-            ]
+                if not menu_item._isDefault and menu_item.is_shown(self.request, self.context)
+            ],
         }, request=self.request)
 
+    def render_default_html(self):
```

Comment:
Could we do this rendering as part of the `render_html` method above?

Reply:
Yes, good catch!